npc/re/quests/quests_rockridge.txt

```diff
@@ -0,0 +1,1741 @@
+		mes "[Shawn McCurdy]";
+		mes "The broken elevator is to the north from the center of this mine.";
+		mes "Check the location of the ores, and bring only ^4d4dff1 Ore Box^000000 with you.";
+		mes "I'll move the rest someplace else.";
+		next;
+		mes "[Shawn McCurdy]";
+		mes "The bandits might stand guard around the mine.";
+		mes "Please be careful.";
+		close;
+	}
+	if (rock_main_quest == 18) {
+		mes "[Shawn McCurdy]";
+		mes "Ah, I should've known";
+		mes "they'd patrol around the elevator.";
+		mes "Get rid of any Buffalo Bandits you see, and get ores.";
+		close;
+	}
+	if (rock_main_quest == 19) {
+		mes "[Shawn McCurdy]";
+		mes "Yes, you brought the correct box.";
+		mes "^4d4dffAlbert Ford^000000 will hit the roof if he sees this.";
+		next;
+		mes "[Shawn McCurdy]";
+		mes "Let's get out of here before Albert finds out why his bandits can't come back from their patrol duty.";
+		mes "Oh, but you first need to ^4d4dfftalk to him.^000000";
+		next;
+		mes "[Shawn McCurdy]";
+		mes "Please ^4d4dffeliminate Buffalo Bandits and take 5 Brown Mufflers and 5 Bandit Scarves with you^000000 on your way there.";
+		next;
+		mes "[Shawn McCurdy]";
+		mes "They'll help create the situation Sheriff Warp wants.";
+		mes "Let's go up, shall we?";
+		erasequest 7806;// Location of the Ores 3
+		setquest 7807;// Ace up Our Sleeve
+		rock_main_quest = 20;
+		close2;
+		warp "rockrdg2",348,214;
+		end;
+	}
+	if (rock_main_quest == 20) {
+		mes "[Shawn McCurdy]";
+		mes "Come on, you need to get rid of Buffalo Bandits and catch Albert off guard!";
+		mes "Up we go!";
+		close2;
+		warp "rockrdg2",348,214;
+		end;
+	}
+	end;
+
+OnInit:
+	questinfo 7801, QTYPE_QUEST, 1;
+	setquestinfo_req 7801,7800,1;
+	end;
+}
+
+rockmi2,97,120,5	script	Broken Elevator#rrevt10	4_ENERGY_WHITE,{
+	if (rock_main_quest < 17) {
+		mes "There's a broken elevator.";
+		close;
+	}
+	if (rock_main_quest == 17) {
+		mes "[????]";
+		mes "Hey look,";
+		mes "a rat has sneaked in.";
+		next;
+		mes "[????]";
+		mes "It's good we've caught it,";
+		mes "isn't it?";
+		close2;
+		rock_main_quest = 18;
+		monster "rockmi2", 95,116,"Buffalo Bandit",3736,1, "Broken Elevator#rrevt10::OnMyMobDead";
+		monster "rockmi2",101,116,"Buffalo Bandit",3737,1, "Broken Elevator#rrevt10::OnMyMobDead";
+		monster "rockmi2", 99,110,"Buffalo Bandit",3738,1, "Broken Elevator#rrevt10::OnMyMobDead";
+		hideonnpc "Broken Elevator#rrevt10";
+		end;
+	}
+	if (rock_main_quest == 18) {
+		mes "I removed the plate from the bottom of the elevator and found boxes of ores underneath it.";
+		unittalk getcharid(3), "" + strcharinfo(0) + " : Shawn said just one box.", bc_self;
+		getitem 25248,1;// Purple Ore Crate
+		rock_main_quest = 19;
+		close;
+	}
+	if (rock_main_quest == 19) {
+		unittalk getcharid(3), "" + strcharinfo(0) + " : I have no more business here. I'd better go back to Shawn.", bc_self;
+		end;
+	}
+	end;
+
+OnMyMobDead:
+	if (mobcount( "rockrdg2", "Broken Elevator#rrevt10::OnMyMobDead" ) < 0)
```

Comment:
Change `if (mobcount( "rockrdg2", "Broken Elevator#rrevt10::OnMyMobDead" ) < 0)`
to `if (mobcount( "rockrdg2", "Broken Elevator#rrevt10::OnMyMobDead" ) == 0)`